public/components/cards/27.html

```diff
@@ -0,0 +1,48 @@
+<a class="block bg-[#fefefe] shadow-xl rounded-xl text-center p-5" href="">
+  <h5
+    class="text-3xl font-bold mb-1 text-transparent bg-clip-text bg-gradient-to-r from-blue-600 via-blue-500 to-blue-600"
+  >
+    Basic Plan
+  </h5>
+
+  <h6 class="text-slate-900 text-sm">
```

Comment:
Add `mt-1` to this element to keep the space.

Throughout HyperUI it uses `text-gray` classes, let's keep that for now but it will be something to look at in the future.